### DISCUSSION TITLE
Preview of block-items in admin UI

### DISCUSSION THREAD

**Discussion by patrickpotthoff** [sulu/sulu#6438] [Ideas]
We mainly use block elements to fill our websites with content. Over time, of course, you have a large number of different block elements. At this point, for example, you don't always know what a block element looks like. 

What do you think about the possibility to show a short preview of the block element? For example in the form of a small magnifying glass icon inside the block, where a mouse over opens a small modal window with a screenshot or something similar to this block. This image could possibly be specified via the .xml as well, similar to <info_text> within the meta information. Of course just optional.

I think that would just help if you don't maintain the website yourself, but the client does. 
I hope you can imagine what I mean.

What do you think about such an idea?

**Comment by ghost** [comment]
We have the same problem and also thought that a preview would be a solution. I will try to implement it. Here is an example from the Neos CMS.

![image](https://user-images.githubusercontent.com/47974249/161697060-9a889c20-fd06-419b-ae85-f5611ba935fd.png)
